packages/contracts-bedrock/scripts/hash-onion/GenerateHashOnion.s.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import { Script } from "forge-std/Script.sol";
+import { console } from "forge-std/Console.sol";
+
+/// @notice Script to generate the hash onion from the tokens json file.
+///         The tokens json file needs to be created on the path specified in the `_tokensPath` function. The keys of
+///         the json must be `localToken` and `remoteToken` with the addresses of the tokens. The
+///         `test-mock-tokens.json` file can be used as a reference (it's only for testing purposes).
+contract GenerateHashOnion is Script {
+    /// @notice Struct to hold the local and remote token addresses.
+    /// @param localToken  Address of the local token.
+    /// @param remoteToken Address of the remote token.
+    struct TokenPair {
+        address localToken;
+        address remoteToken;
+    }
+
+    /// @notice Initial layer of the hash onion.
+    /// keccak256(abi.encode(0));
+    bytes32 internal constant INITIAL_ONION_LAYER = 0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563;
```

Comment:
```suggestion
    bytes32 internal constant INITIAL_ONION_LAYER = keccak256(abi.encode(0));
```
does this work? it's cleaner

Reply:
Hmm I see, I rather not have to trust that the hash is the correct one, right?

Reply:
Perfect, will change it then.
Shall I also change it on the contract where the hash onion feature is implemented?